### PR TITLE
Irminger 7 Recovered Moorings

### PR DESCRIPTION
**Pull request by cdobs**
Creates recovered ingestion csvs for Irminger 7:
-GI01SUMO R00007
-GI02HYPM R00007
-GI03FLMA R00007
-GI03FLMB R00007

### CODE REVIEW

File: GI01SUMO/GI01SUMO_R00007_ingest.csv [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+mi.dataset.driver.dosta_abcdjm.ctdbp_p.dosta_abcdjm_ctdbp_p_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00007/instruments/dcl11/imm/CTDBP_sn16-50158/SBE*.hex,GI01SUMO-RII11-02-DOSTAD033,recovered_inst,Available,
+mi.dataset.driver.dosta_abcdjm.ctdbp_p.dcl.dosta_abcdjm_ctdbp_p_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00007/cg_data/dcl11/imm/ctdbp03/*.DAT,GI01SUMO-RII11-02-DOSTAD033,recovered_host,Available,
+mi.dataset.driver.flord_g.ctdbp_p.dcl.flord_g_ctdbp_p_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00007/cg_data/dcl11/imm/ctdbp01/*.DAT,GI01SUMO-RII11-02-FLORDG031,recovered_host,Available,
+mi.dataset.driver.flord_g.ctdbp_p.dcl.flord_g_ctdbp_p_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00007/cg_data/dcl11/imm/ctdbp02/*.DAT,GI01SUMO-RII11-02-FLORDG032,recovered_host,Available,
+mi.dataset.driver.flord_g.ctdbp_p.dcl.flord_g_ctdbp_p_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00007/cg_data/dcl11/imm/ctdbp03/*.DAT,GI01SUMO-RII11-02-FLORDG033,recovered_host,Available,
```
Comment (snwhite) [inline GI01SUMO/GI01SUMO_R00007_ingest.csv:51]: Should there be lines for FLORD recovered_instr using flord_g_ctdbp_p_recovered_driver?

File: GI01SUMO/GI01SUMO_R00007_ingest.csv [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00007/cg_data/dcl12/hyd2/*hyd2.log,GI01SUMO-SBD12-03-HYDGN0000,recovered_host,Available,
+mi.dataset.driver.pco2a_a.sample.pco2a_a_sample_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00007/cg_data/dcl12/pco2a/*.pco2a.log,GI01SUMO-SBD12-04-PCO2AA000,recovered_host,Available,
+mi.dataset.driver.wavss_a.dcl.wavss_a_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00007/cg_data/dcl12/wavss/*.wavss.log,GI01SUMO-SBD12-05-WAVSSA000,recovered_host,Available,
+mi.dataset.driver.metbk_a.dcl.metbk_a_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00007/cg_data/dcl12/metbk2/*.metbk2.log,GI01SUMO-SBD12-06-METBKA000,recovered_host,Available,
+mi.dataset.driver.fdchp_a.dcl.fdchp_a_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00007/cg_data/dcl12/fdchp/*.fdchp.log,GI01SUMO-SBD12-08-FDCHPA000,recovered_host,Available,
+mi.dataset.driver.fdchp_a.fdchp_a_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00007/instruments/dcl12/FDCHP_sn*/D*/fdchp_*.dat,GI01SUMO-SBD12-08-FDCHPA000,recovered_inst,Available,
```
Comment (snwhite) [inline GI01SUMO/GI01SUMO_R00007_ingest.csv:79]: Looks good.

File: GI02HYPM/GI02HYPM_R00007_ingest.csv [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GI02HYPM/R00007/instruments/CTDMO_sn*/*.hex,GI02HYPM-RIM01-02-CTDMOG039,recovered_inst,Available,
+#mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_recovered_driver,/omc_data/whoi/OMC/GI02HYPM/R00007/instruments/GWFP_sn*/0*/E*.DA?,GI02HYPM-WFP02-00-WFPENG000,recovered_wfp,Pending,unable to parse. Redmine ticket 12792
+mi.dataset.driver.flord_l_wfp.flord_l_wfp_recovered_driver,/omc_data/whoi/OMC/GI02HYPM/R00007/instruments/GWFP_sn*/0*/E*.DA?,GI02HYPM-WFP02-01-FLORDL000,recovered_wfp,Available,
+mi.dataset.driver.dosta_ln.wfp.dosta_ln_wfp_driver,/omc_data/whoi/OMC/GI02HYPM/R00007/instruments/GWFP_sn*/0*/E*.DA?,GI02HYPM-WFP02-03-DOSTAL000,recovered_wfp,Available,
+mi.dataset.driver.ctdpf_ckl.wfp.ctdpf_ckl_wfp_recovered_driver,/omc_data/whoi/OMC/GI02HYPM/R00007/instruments/GWFP_sn*/0*/C*.DA?,GI02HYPM-WFP02-04-CTDPFL000,recovered_wfp,Available,
+mi.dataset.driver.vel3d_l.wfp.vel3d_l_wfp_recovered_driver,/omc_data/whoi/OMC/GI02HYPM/R00007/instruments/GWFP_sn*/0*/A*.DA?,GI02HYPM-WFP02-05-VEL3DL000,recovered_wfp,Available,
```
Comment (snwhite) [inline GI02HYPM/GI02HYPM_R00007_ingest.csv:11]: Looks good.

File: GI03FLMA/GI03FLMA_R00007_ingest.csv [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GI03FLMA/R00007/instruments/CTDMO_sn37-13452/*.hex,GI03FLMA-RIM01-02-CTDMOH051,recovered_inst,Available,
+mi.dataset.driver.sio_eng.sio.sio_eng_sio_recovered_driver,/omc_data/whoi/OMC/GI03FLMA/R00007/cg_data/SSIOC_sn*/STATUS/STA*.DAT,GI03FLMA-RIS01-00-SIOENG000,recovered_host,Available,
+mi.dataset.driver.dosta_abcdjm.sio.dosta_abcdjm_sio_recovered_driver,/omc_data/whoi/OMC/GI03FLMA/R00007/cg_data/SSIOC_sn*/DOSTA/DOS*.DAT,GI03FLMA-RIS01-03-DOSTAD000,recovered_host,Available,
+mi.dataset.driver.phsen_abcdef.phsen_abcdef_recovered_driver,/omc_data/whoi/OMC/GI03FLMA/R00007/instruments/PHSEN_sn*/SAMI*.txt,GI03FLMA-RIS01-04-PHSENF000,recovered_inst,Available,
+mi.dataset.driver.phsen_abcdef.imodem.phsen_abcdef_imodem_recovered_driver,/omc_data/whoi/OMC/GI03FLMA/R00007/cg_data/SSIOC_sn*/PHSEN/PHS*.DAT,GI03FLMA-RIS01-04-PHSENF000,recovered_host,Available,
+mi.dataset.driver.flort_dj.sio.flort_dj_sio_recovered_driver,/omc_data/whoi/OMC/GI03FLMA/R00007/cg_data/SSIOC_sn*/FLORT/FLO*.DAT,GI03FLMA-RIS01-05-FLORTD000,recovered_host,Available,
```
Comment (snwhite) [inline GI03FLMA/GI03FLMA_R00007_ingest.csv:42]: Looks good.

File: GI03FLMB/GI03FLMB_R00007_ingest.csv [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GI03FLMB/R00007/instruments/CTDMO_sn37-13446/*.hex,GI03FLMB-RIM01-02-CTDMOH071,recovered_inst,Available,
+mi.dataset.driver.sio_eng.sio.sio_eng_sio_recovered_driver,/omc_data/whoi/OMC/GI03FLMB/R00007/cg_data/SSIOC_sn*/STATUS/STA*.DAT,GI03FLMB-RIS01-00-SIOENG000,recovered_host,Available,
+mi.dataset.driver.dosta_abcdjm.sio.dosta_abcdjm_sio_recovered_driver,/omc_data/whoi/OMC/GI03FLMB/R00007/cg_data/SSIOC_sn*/DOSTA/DOS*.DAT,GI03FLMB-RIS01-03-DOSTAD000,recovered_host,Available,
+mi.dataset.driver.phsen_abcdef.phsen_abcdef_recovered_driver,/omc_data/whoi/OMC/GI03FLMB/R00007/instruments/PHSEN_sn*/SAMI*.txt,GI03FLMB-RIS01-04-PHSENF000,recovered_inst,Available,
+mi.dataset.driver.phsen_abcdef.imodem.phsen_abcdef_imodem_recovered_driver,/omc_data/whoi/OMC/GI03FLMB/R00007/cg_data/SSIOC_sn*/PHSEN/PHS*.DAT,GI03FLMB-RIS01-04-PHSENF000,recovered_host,Available,
+mi.dataset.driver.flort_dj.sio.flort_dj_sio_recovered_driver,/omc_data/whoi/OMC/GI03FLMB/R00007/cg_data/SSIOC_sn*/FLORT/FLO*.DAT,GI03FLMB-RIS01-05-FLORTD000,recovered_host,Available,
```
Comment (snwhite) [inline GI03FLMB/GI03FLMB_R00007_ingest.csv:42]: Looks good.

**Review by snwhite — APPROVED** [review]

File: GI01SUMO/GI01SUMO_R00007_ingest.csv [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+mi.dataset.driver.flord_g.ctdbp_p.dcl.flord_g_ctdbp_p_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00007/cg_data/dcl11/imm/ctdbp03/*.DAT,GI01SUMO-RII11-02-FLORDG033,recovered_host,Available,
+mi.dataset.driver.pco2w_abc.imodem.pco2w_abc_imodem_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00007/cg_data/dcl11/imm/pco2w1/*.DAT,GI01SUMO-RII11-02-PCO2WC051,recovered_host,Available,
+#mi.dataset.driver.pco2w_abc.pco2w_abc_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00007/instruments/dcl11/imm/PCO2W_snC0103/SAMI*.txt,GI01SUMO-RII11-02-PCO2WC051,recovered_inst,Not Expected,No comms from instrument upon recovery; no data recovered
+mi.dataset.driver.pco2w_abc.imodem.pco2w_abc_imodem_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00007/cg_data/dcl11/imm/pco2w2/*.DAT,GI01SUMO-RII11-02-PCO2WC052,recovered_host,Available,
+mi.dataset.driver.pco2w_abc.pco2w_abc_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00007/instruments/dcl11/imm/PCO2W_snC0104/SAMI*.txt,GI01SUMO-RII11-02-PCO2WC052,recovered_inst,Not Expected,No comms from instrument upon recovery; no data recovered
```
Comment (rtravis-whoi) [inline GI01SUMO/GI01SUMO_R00007_ingest.csv:55]: John & Rebecca - this line should be commented out - no data recovered

**Review by rtravis-whoi — CHANGES_REQUESTED** [review]
Reviewed by John L and Rebecca - One change needed on SUMO ingest - see comment in file.

File: GI01SUMO/GI01SUMO_R00007_ingest.csv [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+mi.dataset.driver.flord_g.ctdbp_p.dcl.flord_g_ctdbp_p_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00007/cg_data/dcl11/imm/ctdbp03/*.DAT,GI01SUMO-RII11-02-FLORDG033,recovered_host,Available,
+mi.dataset.driver.pco2w_abc.imodem.pco2w_abc_imodem_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00007/cg_data/dcl11/imm/pco2w1/*.DAT,GI01SUMO-RII11-02-PCO2WC051,recovered_host,Available,
+#mi.dataset.driver.pco2w_abc.pco2w_abc_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00007/instruments/dcl11/imm/PCO2W_snC0103/SAMI*.txt,GI01SUMO-RII11-02-PCO2WC051,recovered_inst,Not Expected,No comms from instrument upon recovery; no data recovered
+mi.dataset.driver.pco2w_abc.imodem.pco2w_abc_imodem_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00007/cg_data/dcl11/imm/pco2w2/*.DAT,GI01SUMO-RII11-02-PCO2WC052,recovered_host,Available,
+#mi.dataset.driver.pco2w_abc.pco2w_abc_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00007/instruments/dcl11/imm/PCO2W_snC0104/SAMI*.txt,GI01SUMO-RII11-02-PCO2WC052,recovered_inst,Not Expected,No comms from instrument upon recovery; no data recovered
```
Comment (rtravis-whoi) [inline GI01SUMO/GI01SUMO_R00007_ingest.csv:55]: update looks good - line 55 commented out.

**Review by rtravis-whoi — APPROVED** [review]
correction to GI01SUMO_R00007_ingest.csv looks good (commented out line 55).

File: GI01SUMO/GI01SUMO_R00007_ingest.csv (hunk 2 above)
Comment (rtravis-whoi) [reply GI01SUMO/GI01SUMO_R00007_ingest.csv:79]: correction to line 55 good.

File: GI01SUMO/GI01SUMO_R00007_ingest.csv (hunk 7 above)
Comment (snwhite) [reply GI01SUMO/GI01SUMO_R00007_ingest.csv:55]: Looks good.

**Review by snwhite — APPROVED** [review]

**Pull request merged**